V0.5.2BeBuilder/bebuilder.rdef

```diff
@@ -0,0 +1,70 @@
+
+resource app_signature "application/x-vnd.bebuilder";
+
+resource app_version {
+	major  = 0,
+	middle = 0,
+	minor  = 0,
```

Comment:
Also, yeah, I probably could have added the versions there, didn't think about it, but the PR has already been merged and I don't know if it matters all that much.